agones/src/xds.rs

```diff
@@ -358,8 +358,11 @@ management_servers:
             .await
             .unwrap();
         let mount_name = "config";
-        let mut container =
-            quilkin_container(client, Some(vec!["proxy".into()]), Some(mount_name.into()));
+        let mut container = quilkin_container(
```

Comment:
So I think think you'll want/need to change this test.

We will still need a test for the xDS connection, which this will break in this test.

That being said, we'll want a test for direct Agones integration as well. Can probably reuse the RBAC setup functions for this as well.

Reply:
Edited above - I **don't** think think you'll want/need to change this test 🤦🏻 (I can't type)